templates/base.html

```diff
@@ -0,0 +1,27 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.6.2/dist/css/bootstrap.min.css" integrity="sha384-xOolHFLEh07PJGoPkLv1IbcEPTNtaed2xpHsD9ESMhqIYd0nLMwNLD69Npy4HI+N" crossorigin="anonymous">
+    <link rel="stylesheet" >
```

Comment:
I noticed that you used Bootstrap CDN version 4. Next time, consider using the latest available version for updated features and improvements. Additionally, you included a <link rel="stylesheet"> tag, but it appears that you didn't complete all the necessary steps. Specifically, dynamic stylesheets (CSS) weren't added.

>  Don't forget to do all necessary steps so that Django can serve these static files.